oarepo_ui/translations/cs/LC_MESSAGES/messages.po

```diff
@@ -534,3 +534,105 @@ msgstr ""
 #: /Users/m/w/cesnet/oarepo-ui/oarepo_ui/templates/oarepo_ui/header_frontage.html:26
 msgid "Type and press enter to search"
 msgstr "Zadejte a stiskněte Enter pro vyhledávání"
+
+msgid "Community website"
+msgstr "Webové stránky komunity"
+
+msgid "Community selection"
+msgstr "Výběr komunity"
+
+msgid "Please select community in which your work will be published:"
+msgstr "Vyberte prosím komunitu, ve které bude vaše práce publikována:"
+
+msgid "All records must belong to a community."
+msgstr "Všechny záznamy musí patřit komunitě."
+
+msgid "Go back"
+msgstr "Vraťte se"
+
+msgid ""
+"You are not a member of any community. If you choose to proceed, your work "
+"will be published in the \"generic\" community. We strongly recommend that "
+"you join a community to increase the visibility of your work and to "
+"cooperate with others more easily. You can check the available communities "
+"<2>on our communities page.</2> For more details on how to join a community "
+"please refer to the instructions on <6>how to join a community.</6> If you "
+"are certain that you wish to proceed with the generic community, please "
+"click on it below."
+msgstr ""
```

Comment:
@mirekys A lot of these are just translations that are not used anymore, because make-translations, keeps the unused keys that were previously used, will try to remove manually